ProcessMaker/processmaker
# adding environment variables to config horizon processes

Adding environemnt variables make simpler the horizon processes configuration

Variables added:

- PM4_HORIZON_SUPERVISOR_BPMN_PROCESSES
- PM4_HORIZON_SUPERVISOR_BPMN_TRIES
- PM4_HORIZON_SUPERVISOR_BPMN_TIMEOUT
- PM4_HORIZON_SUPERVISOR_1_PROCESSES
- PM4_HORIZON_SUPERVISOR_1_TRIES
- PM4_HORIZON_SUPERVISOR_1_TIMEOUT

## How to Test
Define variables at .env file and check admin/queues route
_If changes is done after application is loaded, the commands *php artisan optimize:clear* && *php artisan horizon:terminate* must be execute to reflect the changes_


## Code Review Checklist
